audiorecorder/src/main/res/raw/isrgrootx1.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This and the `network_security_config` should be in `collect_app` right?

Reply:
So many questions. How did I manage that. How did I not notice. Why did it work. Whut?? 😩